Add session management section (#981)
rancher/rancher-docs: docs/reference-guides/rancher-security/rancher-security-best-practices.md

Adversaries can misuse this information to identify the running Rancher version and cross-relate it with potential bugs to exploit. If your upstream Rancher instance is publicly available on the web, use a Layer 7 firewall to block `/version` and `/rancherversion`.

See [OWASP Web Application Security Testing - Enumerate Infrastructure and Application Admin Interfaces](https://owasp.org/www-project-web-security-testing-guide/stable/4-Web_Application_Security_Testing/02-Configuration_and_Deployment_Management_Testing/05-Enumerate_Infrastructure_and_Application_Admin_Interfaces.html) for more information on protecting your server.

> ### Session management

Reviewer: ```suggestion
### Session Management
```

> Some environments may require extra security controls regarding session management. The upstream Rancher instance does not implement a limit for active sessions for a specific user. 

Reviewer 2: ```suggestion
Some environments may require additional security controls regarding session management, for example, limiting the amount of concurrent active sessions for a given user or restricting where those sessions can be initiated from - to block specific geo-locations or impossible travel incidents. Such features are not supported out-of-the box in Rancher Manager. 
```

Reviewer 3: I like this suggestion ^, would only suggest to simplify and remove the following part - `or impossible travel incidents`. It seems a bit too much for me and the geo-blocking mentioned earlier would already cover this IMO.

> Some environments may require additional security controls for session management. For example, you may want to limit users' concurrent active sessions  or restrict which geolocations those sessions can be initiated from. Such features are not supported by Rancher out of the box. 

Reviewer 3: ```suggestion
Some environments may require additional security controls for session management. For example, you may want to limit users' concurrent active sessions or restrict which geolocations those sessions can be initiated from. Such features are not supported by Rancher out of the box. 
```

> If your Rancher instance is deployed in an environment which requires additional security requirements, use a Layer 7 firewall behind it.

Reviewer 2: ```suggestion
If such features are required, we recommend the combined use of Layer 7 firewalls and [external authentication providers](https://ranchermanager.docs.rancher.com/v2.7/pages-for-subheaders/authentication-config#external-vs-local-authentication).

```

Reviewer 3: Would recommend to link against the latest version, instead of a specific version of the docs -> `https://ranchermanager.docs.rancher.com/pages-for-subheaders/authentication-config#external-vs-local-authentication`.

> If you require such features, combine Layer 7 firewalls with [external authentication providers](../../pages-for-subheaders/authentication-config#external-vs-local-authentication.md).

Reviewer: ```suggestion
If you require such features, combine Layer 7 firewalls with [external authentication providers](../../pages-for-subheaders/authentication-config.md#external-vs-local-authentication).
```
